base/record_replay.cc

```diff
@@ -581,4 +582,44 @@ void DoResetPaintSurface() {
   }
 }
 
+std::vector<std::string> *gPseudoStack = nullptr;
```

Comment:
Why is this in `record_replay.h/cc`? Why not move it to its own file?

Reply:
Bare minimum to get this working.  I don't want to take extra time to make this nice.  I just want _something_ I can use to help me figure out what's calling what in the chromium browser code.

Reply:
"I don't want to take extra time to make this nice."
Moving it to a different file would have definitely saved you time, since changing `record_replay.h` will cause very long compile times 🤔

Reply:
I would have had to update the BUILD files to add it.. but maybe you're right - if google's build scripts are smart enough to incorporate new files without triggering a full rebuild.